D8-launch-tasks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+read -p "Enter the site shortname on ACSF (e.g., \"virtualvisit\", no quotes): " sitename
+# Baby, what time is it?
+timestamp=$(date +%Y%m%d%H%M%S)
+# Generate a random 18-character password that can include mixed-case,
+# alphanumeric, and "_-!.+".
+jamiepass=$(cat /dev/urandom | LC_ALL=C tr -cd a-zA-Z0-9\_\-\!\.\+ | fold -w18 | head -n 1)
+kennedypass=$(cat /dev/urandom | LC_ALL=C tr -cd a-zA-Z0-9\_\-\!\.\+ | fold -w18 | head -n 1)
+morganpass=$(cat /dev/urandom | LC_ALL=C tr -cd a-zA-Z0-9\_\-\!\.\+ | fold -w18 | head -n 1)
+sampass=$(cat /dev/urandom | LC_ALL=C tr -cd a-zA-Z0-9\_\-\!\.\+ | fold -w18 | head -n 1)
+alexpass=$(cat /dev/urandom | LC_ALL=C tr -cd a-zA-Z0-9\_\-\!\.\+ | fold -w18 | head -n 1)
```

Comment:
These are randomized on site creation so this is really belt and suspenders.

Reply:
https://github.com/SU-SWS/stanford_profile/blob/8.x-1.x/src/Plugin/InstallTask/Users.php

Reply:
Ah, GTK.

Reply:
SMRT